lib/internal/loader/CreateDynamicModule.js

```diff
@@ -32,28 +33,39 @@ const createDynamicModule = (exports, url = '', evaluate) => {
   const reflectiveModule = new ModuleWrap(src, `cjs-facade:${url}`);
   reflectiveModule.instantiate();
   const { setExecutor, reflect } = reflectiveModule.evaluate()();
+
+  const importLookup = {};
+  for (const name of imports)
+    importLookup[name] = Buffer.from(name).toString('base64');
```

Comment:
Doesn't base64 also include the characters `+` and `/` which are illegal JS identifiers? Also, a base64 string can start with a digit, which is not allowed in JS identifiers.

You can solve this by replacing `+` with `_`, `/` with `$`, and adding a letter (e.g. `i`) at the beginning of the identifier.

Reply:
ou should also remove the trailing `=` signs that are used for padding base64. Or replace them also with `$`.

Reply:
i already add `$` at the beginning below, but you are right about the other stuff, a huge oversight on my part 😄

Reply:
@devsnek `.toString('hex')`